xilem/src/view/mod.rs

```diff
@@ -32,6 +32,9 @@ mod prose;
 pub use prose::*;
 
 mod textbox;
+mod grid;
+pub use grid::*;
```

Comment:
I'd probably put this after flex, rather than inside `textbox`.

Reply:
Yeah. That didn't make any sense 😄 Probably a merge mistake or IDE automated task.
Fixed.